src/patternfly/utilities/Text/text.scss

```diff
@@ -0,0 +1,77 @@
+// stylelint-disable
+
+// Font size options
+$pf-u-font-size-options: (
+  font-size-xs:  (font-size var(--pf-global--FontSize--xs)),
+  font-size-sm:  (font-size var(--pf-global--FontSize--sm)),
+  font-size-md:  (font-size var(--pf-global--FontSize--md)),
+  font-size-lg:  (font-size var(--pf-global--FontSize--lg)),
+  font-size-xl:  (font-size var(--pf-global--FontSize--xl)),
+  font-size-2xl: (font-size var(--pf-global--FontSize--2xl)),
+  font-size-3xl: (font-size var(--pf-global--FontSize--3xl)),
+  font-size-4xl: (font-size var(--pf-global--FontSize--4xl))
+);
+
+@include pf-utility-builder($pf-u-font-size-options, $pf-global--breakpoint-list);
+
+
+// Font weight options
+$pf-u-font-weight-options: (
+  font-weight-light:              (font-weight var(--pf-global--FontWeight--light)),
+  font-weight-normal:             (font-weight var(--pf-global--FontWeight--normal)),
+  font-weight-bold:               (font-weight var(--pf-global--FontWeight--bold)),
+  font-weight-semi-bold:          (font-weight var(--pf-global--FontWeight--semi-bold)),
+  font-weight-overpass-semi-bold: (font-weight var(--pf-global--FontWeight--overpass--semi-bold)),
+  font-weight-overpass-bold:      (font-weight var(--pf-global--FontSize--2xl))
+);
+
+@include pf-utility-builder($pf-u-font-weight-options, $pf-global--breakpoint-list);
+
+
+// Font color options
+$pf-u-font-color-options: (
+  font-color-100: (color var(--pf-global--Color--100)),
+  font-color-200: (color var(--pf-global--Color--200)),
+  font-color-300: (color var(--pf-global--Color--300)),
+  font-color-400: (color var(--pf-global--Color--400)),
+);
+
+@include pf-utility-builder($pf-u-font-color-options, $pf-global--breakpoint-list);
+
+
+// Background color options
+$pf-u-font-background-color-options: (
+
+  /* Background color */
+  background-color-100:  (background-color var(--pf-global--BackgroundColor--100)),
+  background-color-200:  (background-color var(--pf-global--BackgroundColor--200)),
+
+  /*light theme*/
```

Comment:
maybe we just start with `background-color-100/200` and add the rest if we get specific requests for them.